src/renderer/components/wallet/assets/AssetsTableCollapsable.tsx

```diff
@@ -117,13 +117,13 @@ export const AssetsTableCollapsable: React.FC<Props> = (props): JSX.Element => {
       width: 80,
       render: ({ asset, walletAddress }: WalletBalance) => {
         // Disable UPGRADE button if needed
-        const disableUpgradeButton = disableRuneUpgrade({ asset, haltThorChain, haltEthChain, haltBnbChain })
+        const disableUpgradeButton = disableRuneUpgrade({ asset, haltThorChain, haltEthChain, haltBnbChain, network })
 
         const onClickUpgradeButtonHandler = (e: React.MouseEvent<HTMLElement, MouseEvent>) => {
           e.preventDefault()
           e.stopPropagation()
           setSelectedAsset(O.some(asset))
-          history.push(walletRoutes.upgradeRune.path({ asset: assetToString(asset), walletAddress }))
+          history.push(walletRoutes.upgradeRune.path({ asset: assetToString(asset), walletAddress, network }))
```

Comment:
Same here: No need to pass `network` to `path`

Reply:
Maybe in this case there actually is, since it's being used for `isNonNativeRuneAsset(asset, network)`?

Reply:
No. In `UpgradeView` you can get `network` using `AppContext` if needed. Check other views for a reference.